python/konduit/server.py

```diff
@@ -22,7 +22,8 @@ def stop_server_by_pid(pid):
 class Server(object):
     def __init__(self, inference_config=None, serving_config=None, steps=None,
                  extra_start_args='-Xmx8g', config_path='config.json',
-                 jar_path='konduit.jar'):
+                 jar_path='konduit.jar', pid_file_path='konduit-serving.pid',
```

Comment:
is `pid_file_path` really an argument to this Python object? We create this file server-side, but at least in the current version it's only stored in Python, i.e. if I change the name of the file here, the backend won't be affected.

Reply:
Only thing is that I need to change the location of that pid file while starting that server
I need to override this somehow through the python code
https://github.com/KonduitAI/konduit-serving/blob/3d726a1c377c7753678da012cee2aff158695b0a/konduit-serving-core/src/main/java/ai/konduit/serving/configprovider/KonduitServingNodeConfigurer.java#L120
or maybe get that as info from the server

Reply:
Now you can override the pid file location as well.